### PR TITLE
Improve Standard library dependency update workflow

### PR DESCRIPTION
**Pull request by TharmiganK**

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

**Comment by TharmiganK** [comment]
Will merge this once checked by running the workflow with update 9

**Comment by ThisaruGuruge** [comment]
We can also add `skipGraalvmCheck` flag here.

**Comment by TharmiganK** [comment]
> We can also add `skipGraalvmCheck` flag here.

Yes we can, but I have seen some of the repositories does not have this label(will try to address this in a separate issue). Also we actually don't want to skip this, because lang has no tests with GraalVM compatiblity and between RC packs there can be breaking changes with GraalVM compatiblity. So for safety, we always run this checks but not waiting for them to pass. The release manager/ the module owner can check this manually.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**
